server/app/views/applicant/NorthStarProgramIndexView.java

```diff
@@ -86,6 +92,11 @@ public String render(
     context.setVariable(
         "civicEntityShortName", settingsManifest.getWhitelabelCivicEntityShortName(request).get());
     context.setVariable("sections", sectionParamsBuilder.build());
+    context.setVariable("authProviderName", authProviderName);
+    context.setVariable("loginLink", routes.LoginController.applicantLogin(Optional.empty()).url());
+    context.setVariable("createAccountLink", routes.LoginController.register().url());
```

Comment:
We actually have specs for the header now, which includes the login button https://www.figma.com/file/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?type=design&node-id=1836-7143&mode=design&t=4i1AK4J0bYEHgBBn-0 , so you actually could do it to the specs (although that could be done in a follow up)

The specs show us not having a create account link (because CE login pages have their own create account flow) so you can remove that part (which would also simplify the dom because you wouldn't need the UL and LI elements

Reply:
Note that this is a special case info bar rather than a header, it just seems like a header because we don't have the real header yet. I think the mocks don't make it clear whether this blue info section would have the extra login and create account buttons for guest users like they do in the current design or not. My assumption is it would, as it seems like we really wanted to encourage people to login if they aren't already.

Regardless, definitely for a future change.

Reply:
ok i'm fine leaving it as is until we have more finalized specs for this page